sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseController.java

```diff
@@ -824,44 +824,6 @@ public void selectOrCreateCase(CaseDataDto caseDto, PersonDto person, Consumer<S
 		}
 	}
 
-//	public CommitDiscardWrapperComponent<? extends Component> getCaseCombinedEditComponent(final String caseUuid,
```

Comment:
Please delete commented code

Reply:
That's what I did. Or am I missing something? 😄

Reply:
yeah, sorry. I had the feeling that you've commented the code :D